bots/discord/botify/egg-botify.json

```diff
@@ -0,0 +1,26 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1"
+    },
+    "exported_at": "2020-10-24T16:11:08-03:00",
+    "name": "Botify",
+    "author": "joaquin@rciag.tech",
+    "description": "Discord bot that plays Spotify tracks and YouTube videos or plays any URL including Soundcloud links and Twitch streams.\r\n\r\nTo configure please read: https:\/\/github.com\/robinfriedli\/botify\r\nSteps  1 to 5\r\n\r\nNOTE: YOU NEED A WORKING DATABASE, OTHERWISE IT WONT WORK\r\n\r\nNOTE 2: If you need an updated version you can go to https:\/\/github.com\/robinfriedli\/botify and use those files instead, but you would need to run \"mvn clean install\" as the startup command first and change it back to the original.",
+    "image": "docker.io\/joaquinv98\/maven-pterodactyl:0.0.6",
```

Comment:
It uses maven to compile the bot, the idea is for it to download the base files from github and compile it using maven.
I never made a custom image before, so idk if it is ok but I don't find any maven image that works with pterodactyl, if I'm wrong please let me know and i'll fix it

Reply:
You should only need maven to build it during install. Unless maven is required to run the bot you should just be able to use a java image.

Reply:
I'm not sure, the startup command is a maven command:
**"mvn exec:java -Dexec.mainClass=net.robinfriedli.botify.boot.Launcher"**
Step 6 in the github of the bot: https://github.com/robinfriedli/botify

Reply:
Have you tried without the mvn exec?

Reply:
I tried removing that part and use just "java" instead, with a java img, but it doesn't work, also tried to find a jar file in case there is one so i can try to just execute it but nothing, in botify 2 the actuallly migrate from maven to gradle, and the exec command is just a java -jar command, but this egg is for the original botify

Reply:
Why not build for the latest supported version?

Reply:
It doesn't have any release yet, and i don't use it so i won't be able to tell if it works fine.
2.0 its in development here: [](https://github.com/robinfriedli/botify/tree/development/v2.0)

I'll se if i do it anyways, but they are pretty different bots and botify 1 will keep getting support even after the 2.0 version is released so maybe a different egg for 1.0 and 2.0 will be nice

Reply:
Everything is changed to work with the latest version of botify, the custom docker image for the server is no longer needed